membrane_curvature/core.py

```diff
@@ -5,60 +5,42 @@
 Handles the primary functions
 """
 
-from .lib.mods import dict2pickle, core_fast, curvature
-import sys
-import os
+from numpy.core.fromnumeric import mean
+from .lib.mods import curvature, core_fast_leaflet, gaussian_curvature, mean_curvature
 import time
-import mdtraj as md
 import MDAnalysis as mda
 import math
 from pathlib import Path
 __author__ = "Estefania Barreto-Ojeda"
 version = 0.1
 
-sys.path.append('lib/')
 
 def main():
 
     start_time = time.time()
 
-    # 3. Define leaflets and lipid type
+    # 1. Define leaflets and lipid type # Gone in next steps
     leaflets = ['lower', 'upper']
 
-    #lipid_types = [ ]
-    lipid_types = ['POPC', 'POPE']
-
-    # 4. head indexes
-    head_index = [iil, jjl, jju]
-
-    # 5. Assign topology
-    topology = md.load(grofile).topology
-
-    # 6. Populate universe with coordinates and trajectory
+    # 2. Populate universe with coordinates and trajectory
```

Comment:
MDAnalysis is not restricted to GRO format files. I'd just call it *topology* and *trajectory*.